### ISSUE TITLE
options covered by boundary

### ISSUE THREAD

**Issue by eeyshen** [josdejong/jsoneditor#870]
In autocomplete, when the options menu show above the input, it is open cut by the boundary of the editor, as shown bellow. How to fix it? Thanks.
<img width="140" alt="截屏2019-12-11下午7 53 41" src="https://user-images.githubusercontent.com/31716017/70619479-0aa88880-1c50-11ea-9ed6-6346d517fa83.png">

**Comment by josdejong** [comment]
Thanks for reporting, I can reproduce this issue for example by opening the example examples/11_autocomplete_basic.html, change the size of the editor to a width and height of say 200px. Then start typing.

The reason is that the dropdown menu is relatively positioned, attached to the field. Instead we should make it absolutely positioned on the page. We did this some time ago for the context menu too, so we can probably use that as example.

Anyone able to help out making the dropdown menu absolutely positioned?

**Comment by eeyshen** [comment]
Thanks. I tried the new release. But the bug happens as before.

<img width="144" alt="截屏2020-01-06下午3 32 41" src="https://user-images.githubusercontent.com/31716017/71802793-d3d06180-3099-11ea-8c60-c244b3d3e63e.png">

**Comment by josdejong** [comment]
That is correct, this issue isn't yet resolved. It has a different cause than #869 which was resolved in `v8.1.0` though it probably requires to define a `popupAnchor` as explained in #869.
